packages/mdc-textfield/README.md

```diff
@@ -344,19 +385,31 @@ complicated.
+| removeClassFromBottomLine(className: string) => void | Removes a class from the bottom line element |
 | addClassToHelptext(className: string) => void | Adds a class to the help text element. Note that in our code we check for whether or not we have a help text element and if we don't, we simply return. |
 | removeClassFromHelptext(className: string) => void | Removes a class from the help text element. |
 | helptextHasClass(className: string) => boolean | Returns whether or not the help text element contains the current class |
-| setHelptextAttr(name: string, value: string) => void | Sets an attribute on the help text element |
-| removeHelptextAttr(name: string) => void | Removes an attribute on the help text element |
 | registerInputFocusHandler(handler: EventListener) => void | Registers an event listener on the native input element for a "focus" event |
-| deregisterInputFocusHandler(handler: EventListener) => void | Un-registers an event listener on the native input element for a "focus" event |
+| deregisterInputFocusHandler(handler: EventListener) => void | Deregisters an event listener on the native input element for a "focus" event |
 | registerInputBlurHandler(handler: EventListener) => void | Registers an event listener on the native input element for a "blur" event |
-| deregisterInputBlurHandler(handler: EventListener) => void | Un-registers an event listener on the native input element for a "blur" event |
+| deregisterInputBlurHandler(handler: EventListener) => void | Deregisters an event listener on the native input element for a "blur" event |
 | registerInputInputHandler(handler: EventListener) => void | Registers an event listener on the native input element for an "input" event |
-| deregisterInputInputHandler(handler: EventListener) => void | Un-registers an event listener on the native input element for an "input" event |
+| deregisterInputInputHandler(handler: EventListener) => void | Deregisters an event listener on the native input element for an "input" event |
 | registerInputKeydownHandler(handler: EventListener) => void | Registers an event listener on the native input element for a "keydown" event |
-| deregisterInputKeydownHandler(handler: EventListener) => void | Un-registers an event listener on the native input element for a "keydown" event |
+| deregisterInputKeydownHandler(handler: EventListener) => void | Deregisters an event listener on the native input element for a "keydown" event |
+| registerInputPointerDownHandler(evtType: string, handler: EventListener) => void | Registers an event listener on the native input element for a given event |
+| deregisterInputPointerDownHandler(evtType: string, handler: EventListener) => void | Deregisters an event listener on the native input element for a given event |
+| registerTransitionEndHandler(handler: EventListener) => void | Registers an event listener on the native input element for a "transitionend" event |
```

Comment:
These transitionend APIs aren't actually operating on the native input element as described here; they're registering on the bottom line element if it exists.

Should these be named more specifically to reflect that, in case we ever need to add transitionend handlers for anything else...?

Reply:
I don't think the name should change. It says what the function does. The description should be updated.